data/scenarios/Challenges/_flower-count/judge.sw

```diff
@@ -19,7 +19,7 @@ def judgeCount : Int -> Cmd Unit = \actual.
     (\_. pure ())
     (\p.
       try {
-        let c = (read p : Int) in
+        let c = read @Int p in
         if (c == actual) { win } {}
```

Comment:
I suppose we can't auto convert it, because the type might not have been specified? 🤔